app/services/publish_service_creation.rb

```diff
@@ -163,4 +170,16 @@ def service_csv_output
       deployment_environment: deployment_environment
     ).try(:service_csv_output?)
   end
+
+  def service_email_from
```

Comment:
where is this being used?

Reply:
good spot, forgot to remove in the clean up.